provisioner/roles/manage_ec2_instances/tasks/instances_security_automation.yml

```diff
@@ -1,168 +1,164 @@
-- name: set node types
-  set_fact:
-    node1_node: "{{ ec2_lab_node_types['node1'] }}"
-    node2_node: "{{ ec2_lab_node_types['node2'] }}"
-    node3_node: "{{ ec2_lab_node_types['node3'] }}"
-  when: not networking
-
-- name: find ami for node1
+- name: find ami for splunk
   ec2_ami_facts:
-    region: "{{ ec2_region }}"
-    owners: "{{ ec2_info[node1_node].owners }}"
+    region: "us-east-1" # I don't know why but that seems to be the only place it's available
+    owners: "{{ ec2_info['splunk_enterprise']['owners'] }}"
     filters:
-      name: "{{ ec2_info[ansible_node].filter }}"
-      architecture: "{{ ec2_info[node1_node].architecture }}"
+      name: "{{ ec2_info['splunk_enterprise']['filter'] }}"
+      architecture: "{{ ec2_info['splunk_enterprise']['architecture'] }}"
   register: amis
 
-- name: save ami for node1
+- name: save ami for splunk
```

Comment:
In case of networking we have outsourced the ami part into `ami_find_networking.yml`. Should we do the same with security, moving those bits out of `instances_security_automation.yml` and into a dedicated `ami_find_security.yml`?

I personally would prefer that since to my eyes it makes it much more clean to work with it. But I wouldn't fight for it if you object. What do you think,  @IPvSean and @maxamillion ?

Reply:
yeah I tried to break it up to make it cleaner